lib/rubocop/cop/rspec/multiple_describes.rb

```diff
@@ -22,12 +28,56 @@ module RSpec
       #     describe '.do_something_else' do
       #     end
       #   end
+      #
+      # @example `EnforcedStyle: splitting`
+      #
+      #   # bad
+      #   describe MyClass do
+      #     describe '.do_something' do
+      #     end
+      #     describe '.do_something_else' do
+      #     end
+      #   end
+      #
+      #   # good
+      #   describe MyClass, '.do_something' do
+      #   end
+      #   describe MyClass, '.do_something_else' do
+      #   end
+      #
       class MultipleDescribes < Base
         include TopLevelGroup
+        include ConfigurableEnforcedStyle
 
         MSG = 'Do not use multiple top-level example groups - try to nest them.'
+        MSG_SPLIT = 'Do not use nested describe - try to split them.'
```

Comment:
Example groups have multiple aliases, `example_group`, `describe`, `context`. It's not necessarily `describe`, even though it's the most commonly used.